pkg/tsdb/cloudwatch/time_series_query_test.go

```diff
@@ -136,3 +137,268 @@ func TestTimeSeriesQuery(t *testing.T) {
 		assert.EqualError(t, err, "invalid time range: start time must be before end time")
 	})
 }
+
+func Test_QueryData_executeTimeSeriesQuery_alias_provided_frame_name_uses_period_and_stat_from_expression_when_isUserDefinedSearchExpression(t *testing.T) {
```

Comment:
why not adding these tests in the function above? that way you can have a human readable titles:

`t.Run("alias provided frame uses ...`